Kickstarter-iOS/Views/Controllers/DashboardViewController.swift

```diff
@@ -23,6 +25,10 @@ internal final class DashboardViewController: UITableViewController {
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.tableView.backgroundView = self.backgroundView
+
+    self.tableView.insertSubview(self.loadingIndicatorView, aboveSubview: self.backgroundView)
```

Comment:
Not sure if we do this elsewhere, but we gotta stop :(

From the iOS 11 release notes:

> Adding custom subviews directly to `UITableView` is not supported and might break in a future iOS release. (29094519)

We should start using container views to manage this.

Reply:
(Alternatively we could use a table header view or cell to manage this.)